### ISSUE TITLE
[BUG] Error building version 4.3.4

### ISSUE THREAD

**Issue by bibelin** [dec05eba/gpu-screen-recorder-issues#74]
**Describe the bug**

There is an error while building version 4.3.4 on some distros (probably with not newest libs?)

```
../src/pipewire_audio.c: In function ‘gsr_pipewire_create_null_audio_sink’:
../src/pipewire_audio.c:373:31: error: storage size of ‘err_loc’ isn’t known
  373 |     struct spa_error_location err_loc;
```

**To Reproduce**

Building GSR on stable Debian, Ubuntu or Fedora <= 40.

**Additional context**

https://github.com/pacstall/pacstall-programs/pull/6738
https://copr.fedorainfracloud.org/coprs/brycensranch/gpu-screen-recorder-git/build/8363857/

**Comment by dec05eba** [comment]
Can you test build on latest master? Im not sure if its just a missing include or if the api is different in different versions

**Comment by bibelin** [comment]
Tried master, no changes, same error.

**Comment by dec05eba** [comment]
I see, so the api has changed. I removed the code, it's not really needed. Can you try building again on the latest master?

**Comment by bibelin** [comment]
Fixed now, thanks :)

**Comment by dec05eba** [comment]
Thanks, i'll released version 4.3.5 with the fix
